packages/trader/src/Modules/SmartChart/Containers/smart-chart.jsx

```diff
@@ -69,7 +69,7 @@ class Chart extends React.Component {
                 shouldFetchTradingTimes={!this.props.end_epoch}
                 refreshActiveSymbols={this.props.should_refresh_active_symbols}
             >
-                { this.props.markers_array.map((marker, idx) => (
+                {this.props.is_chart_ready && this.props.markers_array.map((marker, idx) => (
```

Comment:
@nazanin-binary i think this check is no longer necessary since you've handled it in SmartCharts here: https://github.com/binary-com/SmartCharts/pull/830/files#diff-cc09b7152c6a648c48bb7ff59e510e41R285

Reply:
@kelly-binary that is for the Smartcharts demo app, that doesn't affect Deriv

Reply:
@nazanin-binary okay, do you need this PR to be merged?

Reply:
yes it's part of the,but it can be merged sooner/later, they do not depend on each other :-D

Reply:
@nazanin-binary IMO, it'd be better to move this condition to smartchart in `Chart.jsx` or in `marker-store` where the chart is decided to draw the marker or not.